example/src/screens/CreateRoom.tsx

```diff
@@ -0,0 +1,267 @@
+import { BackgroundWrapper } from '@components/BackgroundWrapper';
+import { Modal } from '@components/Modal';
+import { TextInput } from '@components/TextInput';
+import { Typo } from '@components/Typo';
+import { StandardButton } from '@components/buttons/StandardButton';
+import { SERVER_URL } from '@env';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import { RootStack } from '@model/NavigationTypes';
+import { useHeaderHeight } from '@react-navigation/elements';
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React, { useEffect, useState, useCallback } from 'react';
+import {
+  Alert,
+  View,
+  StyleSheet,
+  KeyboardAvoidingView,
+  Platform,
+  PermissionsAndroid,
+  ScrollView,
+} from 'react-native';
+import { useVideoroomState } from 'src/VideoroomContext';
+
+function isEmpty(value) {
```

Comment:
it looks like it might be useful in the future, maybe extract it to `@utils` or something
you can also add a library like `lodash` or whatever you're familiar with/is popular right now

Reply:
using loadsh now.